src/sentry/testutils/cases.py

```diff
@@ -3255,6 +3255,11 @@ def create_span(
 
 
 class TraceTestCase(SpanTestCase):
+    def setUp(self):
+        self.day_ago = before_now(days=1).replace(hour=10, minute=0, second=0, microsecond=0)
+        self.root_span_ids = [uuid4().hex[:16] for _ in range(3)]
+        self.trace_id = uuid4().hex
```

Comment:
I move this logic here since it will be setUp for each of my tests for my endpoint.